src/synthetic_population_generator/cli.py

```
@@ -235,6 +277,93 @@ def filter_data(ctx, input_csv_path, output_file, conditions):
+def fetch_abs_data(ctx, dataflow_id, output_file, agency_id, series_key,
+                   start_period, end_period, updated_after,
+                   first_n_observations, last_n_observations,
+                   dimension_at_observation, detail, output_format):
+    """
+    Fetches data from a specified Australian Bureau of Statistics (ABS) dataflow.
+    """
+    config = ctx.obj['CONFIG']
+    logger.info(f"Executing 'fetch-abs-data' for dataflow '{dataflow_id}'. Output: {output_file}, Format: {output_format}")
+
+    abs_query_params = {}
+    if start_period: abs_query_params['startPeriod'] = start_period
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)